registry/lib/components/style/custom-navbar/index.ts

```diff
@@ -91,6 +92,11 @@ const options = defineOptionsMetadata({
     dropdownEnum: NavbarNotifyStyle,
     displayName: '消息提醒样式',
   },
+  linkPopupContentAlignStyle: {
+    defaultValue: NavbarLinkPopupContentAlignStyle.Left,
+    dropdownEnum: NavbarLinkPopupContentAlignStyle,
+    displayName: '仅包含链接的导航栏弹窗文字对齐样式',
```

Comment:
这里简单点写就行, 我后续在描述里加点详细说明
```suggestion
    displayName: '链接对齐样式',
```